src/containers/AdminCampaignEdit/sections/CampaignCannedResponsesForm/index.tsx

```diff
@@ -133,7 +133,7 @@ class CampaignCannedResponsesForm extends React.Component<InnerProps, State> {
     const cannedResponsesToAdd = this.state.cannedResponsesToAdd.concat({
       ...response,
       id: newId,
-      displayOrder: cannedResponses.length + 1
+      displayOrder: cannedResponses.length
```

Comment:
question: does this result in two `cannedResponses` with the same display order?

Reply:
We moved from 1-based indexing to 0-based indexing after feedback in #1540 so this was a holdover from that which was really throwing me off while debugging for a bit 😭